index.js

```diff
@@ -404,19 +395,13 @@ class Analytics {
       });
   }
 
-  _isErrorRetryable(error) {
-    // Retry Network Errors.
-    if (axiosRetry.isNetworkError(error)) {
+  _isErrorRetryable(attempt, error, response) {
+    // retry on any network error
+    if (error !== null) {
       return true;
     }
 
-    if (!error.response) {
-      // Cannot determine if the request can be retried
-      return false;
-    }
-
-    this.logger.error("error status: " + error.response.status);
-    // Retry Server Errors (5xx).
+    // retry on 5xx status codes
     if (error.response.status >= 500 && error.response.status <= 599) {
```

Comment:
Use response (is error.response even defined?)
```suggestion
    if (response.status >= 500 && response.status <= 599) {
```